### PR TITLE
feat: use biome check

### LINKED ISSUE #23: Support for 'organizeImports'
Thank you for the plugin! I'm using v1-nightly, along with Biome 1.5.1.

In my biome.json file, I have enabled organizeImports:
```json
  "organizeImports": {
    "enabled": true
  }
```
However, neither the safe fix nor the formatting commands have any effect on the import order. Am I missing something, or is this not supported yet?

### PR DESCRIPTION
**Pull request by victor-teles**
# Summary

This PR closes #23
Now the plugin run `biome check` instead of `biome format` and `biome lint` in separated form.

This PR also implement  a improvement to the auto-save configuration description:
CleanShot 2024-01-14 at 01 37 54@2x

When the biome is not found

CleanShot 2024-01-14 at 01 45 48@2x

### CODE REVIEW

File: src/main/kotlin/com/github/biomejs/intellijbiome/BiomeStdinRunner.kt [hunk 1]
```diff
@@ -17,54 +18,17 @@ import kotlin.time.Duration
...
+        val commandLine = createCommandLine(request.virtualFile, "check", getCheckFlags(features))
         val file = request.virtualFile
         val timeout = request.timeout
-        val future = startTheFuture(
-            BiomeBundle.message(
-                "biome.failed.to.format.file",
-                file.name
-            ), timeout
+        val failureMessage = BiomeBundle.message(
+            "biome.failted.to.run.biome.check.with.features",
```
Comment (oferitz) [inline src/main/kotlin/com/github/biomejs/intellijbiome/BiomeStdinRunner.kt:26]: A tiny typo it should be `"biome.failed.to.run.biome.check.with.features"`

File: src/main/resources/messages/BiomeBundle.properties [hunk 2]
```diff
@@ -1,30 +1,23 @@
...
-action.ReformatWithBiomeAction.description=Reformat with Biome
-biome.apply.safe.fix.with.biome=Apply safe fixes with Biome
-biome.apply.unsafe.fix.with.biome=Apply unsafe fixes with Biome
-biome.failed.to.format.file=Biome failed to format the file
-biome.failed.to.fix.file=Biome failed to fix the file
+biome.run.format.on.save.checkbox.on.actions.on.save.page=Run Biome check
+biome.run.safe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome check with --apply
+biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome  check with --apply-unsafe
+biome.run.biome.check.with.features=Running Biome check with {0}
+biome.failted.to.run.biome.check.with.features=Failed to run Biome check with {0} for file {1}
```
Comment (oferitz) [inline src/main/resources/messages/BiomeBundle.properties:20]: A tiny typo it should be `biome.failed.to.run.biome.check.with.features`

File: src/main/kotlin/com/github/biomejs/intellijbiome/BiomeStdinRunner.kt (hunk 1 above)
Comment (victor-teles) [reply src/main/kotlin/com/github/biomejs/intellijbiome/BiomeStdinRunner.kt:26]: Nice catch! Thank you

File: src/main/resources/messages/BiomeBundle.properties (hunk 2 above)
Comment (BC-M) [reply src/main/resources/messages/BiomeBundle.properties:20]: In `Run Biome  check with --apply-unsafe` is an double space right after `Run Biome`.

**Review by BC-M — CHANGES_REQUESTED** [review]

File: src/main/resources/messages/BiomeBundle.properties [hunk 3]
```diff
@@ -1,30 +1,23 @@
...
-biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome  lint --apply-unsafe
-action.ReformatWithBiomeAction.text=Reformat with Biome
-action.ReformatWithBiomeAction.description=Reformat with Biome
-biome.apply.safe.fix.with.biome=Apply safe fixes with Biome
-biome.apply.unsafe.fix.with.biome=Apply unsafe fixes with Biome
-biome.failed.to.format.file=Biome failed to format the file
-biome.failed.to.fix.file=Biome failed to fix the file
+biome.run.format.on.save.checkbox.on.actions.on.save.page=Run Biome check
+biome.run.safe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome check with --apply
+biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page=Run Biome  check with --apply-unsafe
```
Comment (BC-M) [inline src/main/resources/messages/BiomeBundle.properties:18]: Here is a double space after `Run Biome` in `biome.run.unsafe.fixes.on.save.checkbox.on.actions.on.save.page`.

Comment (victor-teles) [reply src/main/resources/messages/BiomeBundle.properties:18]: Nice, thank you

**Comment by BC-M** [comment]
If it fits your schedule, I would be happy about a fresh nightly release. We are currently migrating from Prettier to Biome and are already testing the latest nightly version - "biome check" would be a nice addition here.

**Comment by victor-teles** [comment]
@BC-M  Done! let me know what do you think about the new version!
I really want to release a stable version very soon, any feedbacks about the nightly version are welcome ✌🏻

**Comment by oferitz** [comment]
@victor-teles Thanks! downloaded the latest nightly and it's working great so far.

**Pull request merged**